server/protocols/jmap/src/main/java/org/apache/james/jmap/json/ObjectMapperFactory.java

```diff
@@ -56,10 +59,19 @@
 
 public class ObjectMapperFactory {
 
-    private static final ImmutableSet.Builder<Module> JACKSON_BASE_MODULES = ImmutableSet.<Module>builder().add(
+    private static final ImmutableSet.Builder<Module> JACKSON_BASE_MODULES;
+
+    static {
+        JavaTimeModule javaTimeModule = new JavaTimeModule();
+
+        javaTimeModule.addDeserializer(Instant.class, InstantDeserializer.INSTANT);
+        javaTimeModule.addSerializer(Instant.class, InstantSerializer.INSTANCE);
+
+        JACKSON_BASE_MODULES = ImmutableSet.<Module>builder().add(
             new Jdk8Module(),
             new JavaTimeModule(),
```

Comment:
You're not using the javaTimeModule from above, that you've been wrapping in a static bloc for this?

Also static blocs are evil, please prefer a static method...